pkg/lib/keptn/keptn_base.go

```diff
@@ -216,10 +216,10 @@ func ValidateKeptnEntityName(name string) bool {
 	return len(processedString) == len(name)
 }
 
-// ValididateUnixDirectoryName checks whether the provided dirName contains
+// ValidateUnixDirectoryName checks whether the provided dirName contains
 // any special character according to
 // https://www.cyberciti.biz/faq/linuxunix-rules-for-naming-file-and-directory-names/
-func ValididateUnixDirectoryName(dirName string) bool {
+func ValidateUnixDirectoryName(dirName string) bool {
```

Comment:
This is an API break -- https://github.com/keptn/keptn/pull/3234#discussion_r573863446

It probably makes sense to split this off into its own PR. But, I'm leaving it here until someone provides feedback.

Reply:
Hi @jsoref, I agree with you! Please move this change into another PR. 

Other than that, this PR looks good. Thanks!